app/Providers/MugShotServiceProvider.php

```diff
@@ -25,6 +25,9 @@ public function register()
                 ->userAgent($config['request']['useragent'])
                 ->ignoreHttpsErrors();
 
+            if ( $config['puppeteer']['noSandbox']) {
```

Comment:
As I already wrote above: I would swap that to:
```suggestion
            if (!$config['puppeteer']['sandbox']) {
```
Makes it harder to read in the service provider, but easier for the env and the config file

Reply:
right! fixed